scheduler/minimesosFile

```diff
@@ -39,7 +39,7 @@ minimesos {
         imageName = "containersol/mesos-agent"
         imageTag = "1.0.0-0.1.0"
         loggingLevel = "# INHERIT FROM CLUSTER"
-        portNumber = 5052
+        portNumber = 5051
```

Comment:
I think it works becuase the agents are using a different IP.

Also, I see the folllowing message on my mac:
`Detected Mac Environment X, running with '--mapPortsToHost' so master and marathon ports are mapped to localhost`